v1/src/components/BynderImageField.tsx

```diff
@@ -1,99 +1,124 @@
-import {contentItemMethods, openModal, useAgilityAppSDK, useResizeHeight} from "@agility/app-sdk"
-import {useMemo} from "react"
-import EmptySection from "./EmptySection"
-import AttachmentOverlay from "./AttachmentOverlay"
-import Metadata from "./Metadata"
-import {IconBuildingStore, IconPhoto, IconPhotoOff} from "@tabler/icons-react"
-import {Button} from "@agility/plenum-ui"
-import FieldHeader from "./FieldHeader"
+import {
+  contentItemMethods,
+  openModal,
+  useAgilityAppSDK,
+  useResizeHeight,
+} from "@agility/app-sdk";
+import { useMemo } from "react";
+import EmptySection from "./EmptySection";
+import AttachmentOverlay from "./AttachmentOverlay";
+import Metadata from "./Metadata";
+import {
+  IconBuildingStore,
+  IconPhoto,
+  IconPhotoOff,
+} from "@tabler/icons-react";
+import { Button } from "@agility/plenum-ui";
+import FieldHeader from "./FieldHeader";
 
 export default function BynderImageField() {
-	const {initializing, appInstallContext, field, fieldValue} = useAgilityAppSDK()
+  const { initializing, appInstallContext, field, fieldValue } =
+    useAgilityAppSDK();
 
-	const ref = useResizeHeight()
+  const ref = useResizeHeight();
```

Comment:
This const can be more descriptive 

`const resizeHeightRef = useResizeHeight();`